Resources/Prototypes/Entities/Structures/Walls/walls.yml

```diff
@@ -241,36 +241,6 @@
     key: walls
     base: ice
 
-- type: entity
-  parent: WallBase
-  id: WallMetal
```

Comment:
You removed normal walls?

Reply:
Normal walls are solid walls, metal walls have been deprecated forever

Reply:
Your gonna need to fix the maps if you want to kill this

Reply:
There wasn't any issues at time of PR. If it's conflicting now that's the price of keeping up with map merges :P will check.